grakn-factory/src/main/java/ai/grakn/factory/TxFactoryJanusHadoop.java

```diff
@@ -43,15 +44,22 @@
  * @author fppt
  */
 public class TxFactoryJanusHadoop extends TxFactoryAbstract<EmbeddedGraknTx<HadoopGraph>, HadoopGraph> {
-    private static final String CLUSTER_KEYSPACE = "janusmr.ioformat.conf.storage.cassandra.keyspace";
-    private static final String INPUT_KEYSPACE = "cassandra.input.keyspace";
+    private static final String JANUSMR_IOFORMAT_CONF_STORAGE_CASSANDRA_KEYSPACE = "janusmr.ioformat.conf.storage.cassandra.keyspace";
+    private static final String JANUSMR_IOFORMAT_CONF_STORAGE_HOSTNAME = "janusmr.ioformat.conf.storage.hostname";
+    private static final String JANUSGRAPHMR_IOFORMAT_CONF_STORAGE_HOSTNAME = "janusgraphmr.ioformat.conf.storage.hostname";
+    private static final String JANUSGRAPHMR_IOFORMAT_CONF_STORAGE_CASSANDRA_KEYSPACE = "janusgraphmr.ioformat.conf.storage.cassandra.keyspace";
+    private static final String CASSANDRA_INPUT_KEYSPACE = "cassandra.input.keyspace";
+
     private final Logger LOG = LoggerFactory.getLogger(TxFactoryJanusHadoop.class);
 
     TxFactoryJanusHadoop(EmbeddedGraknSession session) {
         super(session);
 
-        session().config().properties().setProperty(CLUSTER_KEYSPACE, session().keyspace().getValue());
-        session().config().properties().setProperty(INPUT_KEYSPACE, session().keyspace().getValue());
+        session().config().properties().setProperty(JANUSMR_IOFORMAT_CONF_STORAGE_CASSANDRA_KEYSPACE, session().keyspace().getValue());
+        session().config().properties().setProperty(CASSANDRA_INPUT_KEYSPACE, session().keyspace().getValue());
+        session().config().properties().setProperty(JANUSGRAPHMR_IOFORMAT_CONF_STORAGE_HOSTNAME, session().config().getProperty(GraknConfigKey.STORAGE_HOSTNAME));
+        session().config().properties().setProperty(JANUSMR_IOFORMAT_CONF_STORAGE_HOSTNAME, session().config().getProperty(GraknConfigKey.STORAGE_HOSTNAME));
+        session().config().properties().setProperty(JANUSGRAPHMR_IOFORMAT_CONF_STORAGE_CASSANDRA_KEYSPACE, session().keyspace().getValue());
```

Comment:
Again probably worth introducing something like the `overrideMap` as in `TxFactoryJanus`. Would make things clearer.